docs/design/vocabulary.md

```diff
@@ -120,15 +172,16 @@ vocabularyManager.addVocabulary(enVocString);
 Use the `getTerm` method on the `VocabularyManager` to retrieve a term for
 a declaration or property within a namespace:
 
-
 ```
-const term = vocabularyManager.getTerm('org.acme', 'en-gb', 'Color');
-// term.should.equal('A colour');
+const shortTerm = vocabularyManager.getTerm('org.acme', 'en-gb', 'Color', { termType: 'short' });
```

Comment:
```suggestion
const shortTerm = vocabularyManager.getTerm('org.acme', 'en-gb', 'Color', 'short');
```

Reply:
API signature is different from the one mentioned here in this change. You may want to update at other places as well.